blocks/init/src/Blocks/components/button/button.php

```diff
@@ -3,11 +3,12 @@
 /**
  * Template for the Button Component.
  *
- * @package EightshiftBoilerplate
+ * @package Project
  */
 
-use EightshiftBoilerplateVendor\EightshiftLibs\Helpers\Components;
+use ProjectVendor\EightshiftLibs\Helpers\Components;
```

Comment:
Same question with this one. I thought this gets changed with WPCLI scripts.

Reply:
she same as up